Watchman.Discord/IoC/Modules/ServiceModule.cs

```diff
@@ -16,7 +17,7 @@ public class ServiceModule : Autofac.Module
     {
         protected override void Load(ContainerBuilder builder)
         {
-            builder.Register((c, p) => new ResponsesService().SetGetResponsesFromDatabase(c.Resolve<IQueryBus>()))
+            builder.Register((c, p) => new ResponsesCachingService(c.Resolve<DiscordServersService>()).SetGetResponsesFromDatabase(c.Resolve<IQueryBus>()))
```

Comment:
`SetGetResponsesFromDatabase`

fajnie gdyby w IoC nie było takich krzywych obejść bo później ktoś będzie długo sie zastanawiać "skąd to sie tu wzięło"